src/pages/settings/tabs/UserManagementSettings.tsx

```diff
@@ -67,15 +70,15 @@ function UserManagementSettings() {
   const tags = useGetAniDBTagsQuery({ pageSize: 0, excludeDescriptions: true });
 
   useEffect(() => {
-    if (users.length > 0) setSelectedUser(users[0]);
+    if (users.length > 0) setSelectedUser(cloneDeep(users[0]));
```

Comment:
Because otherwise we may be editing the data from the user list? Is it wrong to do a deep clone here?

Reply:
Should I revert it?

Reply:
no actual need to clone, you only need to clone if you edit the array, but we also have immer so it is preferable to use that instead of cloning, as immer will take care of all the needed cloning automatically and prevent any errors in that regards, but here you don't need a deepClone.

Reply:
Data from `users` cannot really be edited but I get your point.
Also, we should probably change `selectedUser` to use Immer as avael said